grails-app/services/org/bbop/apollo/ChadoHandlerService.groovy

```diff
@@ -33,10 +33,12 @@ class ChadoHandlerService {
     def transcriptService
     def cdsService
 
-    private static final SEQUENCE_ONTOLOGY = "sequence"
-    private static final RELATIONSHIP_ONTOLOGY = "relationship"
-    private static final FEATURE_PROPERTY = "feature_property"
-
+    private static final String SEQUENCE_ONTOLOGY = "sequence"
```

Comment:
at some point we might want to move these to a ChadoProepertyEnum, but I think that this is good for now.